app/assets/stylesheets/common/base/login.scss

```diff
@@ -1,3 +1,14 @@
+$invite-bg: svg-uri(
+  '<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Seems like this is causing errors with the CI build:


```
ERROR Summary:

  - broccoliBuilderErrorStack: Error: Expected '.
  ╷
2 │   '<?xml version="1.0" encoding="UTF-8"?>
  │                                          ^
  ╵
  ../../stylesheets/common/base/login.scss 2:42   @import
  ../../stylesheets/common/base/_index.scss 31:9  @import
  ../../stylesheets/common.scss 15:9              @import
  ../../stylesheets/desktop.scss 1:9              @import
  ../../stylesheets/testem.scss 20:9              root stylesheet
    at Object.wrapException (/__w/discourse/discourse/app/assets/javascripts/node_modules/sass/sass.dart.js:1247:17)
    at Object.throwWithTrace0 (/__w/discourse/discourse/app/assets/javascripts/node_modules/sass/sass.dart.js:23147:15)
    at ScssParser0.wrapSpanFormatException$1$1 (/__w/discourse/discourse/app/assets/javascripts/node_modules/sass/sass.dart.js:90289:13)
    at ScssParser0.wrapSpanFormatException$1 (/__w/discourse/discourse/app/assets/javascripts/node_modules/sass/sass.dart.js:90295:19)
```

Not really sure why since we do a similar thing here?

https://github.com/discourse/discourse/blob/4c085873e5c4174e876304d8300462fb96a7fd6e/app/assets/stylesheets/wizard.scss#L1-L10